latest/schemas/image.schema

```diff
@@ -1,229 +1,348 @@
-            "enum": [
-              "scale"
-            ]
+    "TranslationTransform": {
+      "properties": {
+        "type": {
+          "const": "translation",
+          "enum": [
+            "translation"
+          ],
+          "title": "Type",
+          "type": "string"
+        },
+        "translation": {
+          "items": {
+            "type": "number"
           },
-          "scale": {
-            "type": "array",
-            "minItems": 2,
-            "items": {
-              "type": "number"
-            }
-          }
+          "minItems": 1,
+          "title": "Translation",
+          "type": "array"
         }
       },
-      "maxContains": 1,
+      "required": [
+        "type",
+        "translation"
+      ],
+      "title": "TranslationTransform",
+      "type": "object"
+    },
+    "Window": {
+      "properties": {
+        "start": {
+          "title": "Start",
+          "type": "number"
+        },
+        "end": {
+          "title": "End",
+          "type": "number"
+        },
+        "min": {
+          "title": "Min",
+          "type": "number"
+        },
+        "max": {
+          "title": "Max",
+          "type": "number"
+        }
+      },
+      "required": [
+        "start",
+        "end",
+        "min",
+        "max"
+      ],
+      "title": "Window",
+      "type": "object"
+    }
+  },
+  "description": "JSON from OME-NGFF .zattrs",
+  "properties": {
+    "multiscales": {
       "items": {
-        "oneOf": [
-          {
-            "type": "object",
-            "properties": {
-              "type": {
-                "type": "string",
-                "enum": [
-                  "scale"
-                ]
-              },
-              "scale": {
-                "type": "array",
-                "minItems": 2,
-                "items": {
-                  "type": "number"
-                }
-              }
-            },
-            "required": ["type", "scale"]
-          },
-          {
-            "type": "object",
-            "properties": {
-              "type": {
-                "type": "string",
-                "enum": [
-                  "translation"
-                ]
-              },
-              "translation": {
-                "type": "array",
-                "minItems": 2,
-                "items": {
-                  "type": "number"
-                }
-              }
-            },
-            "required": ["type", "translation"]
-          }
-        ]
-      }
+        "$ref": "#/$defs/Multiscale"
+      },
+      "minItems": 1,
+      "title": "Multiscales",
+      "type": "array"
+    },
+    "omero": {
+      "anyOf": [
+        {
+          "$ref": "#/$defs/Omero"
+        },
+        {
+          "type": "null"
+        }
+      ],
+      "default": null
     }
-  }
+  },
+  "required": [
+    "multiscales"
+  ],
+  "title": "NGFF Image",
+  "type": "object",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
@d-v-b: did this get run through a prettifier? What's the minimal diff?

Reply:
I generated these schemas with pydantic models (deleted in [this commit](https://github.com/ome/ngff/pull/235/commits/aa5c9532d1155bf3040d7bdf951ae63fab8aa560)), What's the particular issue here?

Reply:
@d-v-b The textual diff is enormous/all-encompassing so it's hard for a reviewer to tell what's changed.

Potential suggestion: make the equivalent pydantic model for the previous spec, generate it (and make sure that it's not semantically different from the main branch), then update it with these changes, then we should be able to see the minimal diff in that commit.

Of course, if there *are* semantic differences, it might be worth (a) checking whether they are real or bugs, and (b) making a separate PR to update them, if needed.

Reply:
gotcha, I will see what I can do to clean up the diff

Reply:
and while we're on the subject, what do we think about renaming the schema files to end with `.schema.json` instead of `.schema`, since they are json?

Reply:
Hmm, we actually weren't on the subject. 😂 My suggestion is to raise that in a different issue/PR.

Reply:
diff should be much cleaner now

Reply:
Thanks, @d-v-b. :+1: for ginned up clean ups post-RFC-3.